Client/Frontend/Widgets/TabsButton.swift

```diff
@@ -141,6 +152,23 @@ extension TabsButton: Themeable {
         titleBackgroundColor = theme.backgroundColor
         textColor = theme.textColor
         insets = theme.insets!
+
+        if themeName == Theme.NormalMode || themeName == Theme.PrivateMode {
+            previousTheme = themeName
+        }
+    }
+
+    func applyHighlight(theme: String) {
+        if theme == Theme.NormalMode {
+            borderColor = TabsButtonUX.TitleBackgroundColor
+            titleBackgroundColor = TabsButtonUX.TitleColor
+            textColor = TabsButtonUX.TitleBackgroundColor
+        } else {
+            borderColor = UIConstants.PrivateModePurple
+            titleBackgroundColor = UIConstants.PrivateModePurple
+            textColor =  TabsButtonUX.TitleColor
+        }
```

Comment:
Doing a comparison on the current theme and then setting the UI based on that theme is exactly what the `Theme` class is supposed to do. Add these properties directly into `Theme` -- then we can just do `borderColor = theme.TabsHighlightBorderColor`, etc.